neuralmonkey/encoders/vanilla_sentence_encoder.py

```diff
@@ -0,0 +1,105 @@
+import tensorflow as tf
+import numpy as np
+
+from neuralmonkey.logging import log
+from neuralmonkey.nn.bidirectional_rnn_layer import BidirectionalRNNLayer
+from neuralmonkey.checking import assert_type
+from neuralmonkey.decoding_function import Attention
+from neuralmonkey.vocabulary import Vocabulary
+
+# tests: mypy
+
+class VanillaSentenceEncoder(object):
+    """Like the SentenceEncoder, but ignores the start/end tokens.
```

Comment:
Já bych to možná celý zahodil ten vanilla encoder.. My jsme nevěděli, jestli to náhodou neni bug. Ale pak se to učilo nějak debilně.. Nevíš, jestli se do enkodéru ty start a end tokeny maj nebo nemaj dávat? (Default co tam máme od začátku je že tam jsou, ale já necejtim moc důvod)